src/mantine-dates/src/components/TimeRangeInput/demos/index.ts

```diff
@@ -1,2 +1,3 @@
 export { usage } from './usage';
+export { configurator } from './configurator';
```

Comment:
There is no need to include configurator for TimeRangeInput, docs will be merged with TimeInput in one of future release